### PR TITLE
modified the run.sh and file path variables

### PR DESCRIPTION
**Pull request by prashanthShiksha**

### CODE REVIEW

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh [hunk 1]
```diff
@@ -4,31 +4,31 @@
...
 ## It uses curl for API requests and jq to handle JSON responses.
 
+#External Path 
+report_path=$1
+folder_name=$(basename "$report_path")
+
+new_name=$(echo "$folder_name" | tr -s '-' ' ')
+
 # Color codes for terminal output
 BOLD_CYAN="\033[1;36m"
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh:14]: BOLD_CYAN is not used anywhere in the script

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh [hunk 2]
```diff
@@ -4,31 +4,31 @@
...
+
 # Metabase server URL and credentials
 METABASE_URL="http://localhost:3000"
 METABASE_USERNAME="analytics@shikshalokam.org"
 METABASE_PASSWORD="analytics@123"
 
 # Collection and dashboard details
-COLLECTION_NAME="Improvement Projects"
-DASHBOARD_NAME="Improvement Projects Report"
+COLLECTION_NAME=$(echo "$new_name" | sed 's/ Report$//')
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh:27]: In the same line can you replace Report with Collection

File: metabase/metabase-scripts/elevate-projects/02_get_table_data.sh [hunk 3]
```diff
@@ -11,8 +11,10 @@ NC="\033[0m"
 echo -e "${BOLD_YELLOW}          :: Get the required table data from Metabase ::              ${NC}"
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/02_get_table_data.sh:11]: BOLD_GREEN is not used anywhere in this script remove that

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh [hunk 4]
```diff
@@ -136,6 +136,8 @@ echo ">>  [01_create_dashboard.sh] Script executed successfully!"
 echo ""
 echo ""
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh:137]: remove one empty echo line

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh [hunk 5]
```diff
@@ -136,6 +136,8 @@ echo ">>  [01_create_dashboard.sh] Script executed successfully!"
...
 sleep 2
-
+echo "$pwd"
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh:139]: what is the use of line 139 and 141

File: metabase/metabase-scripts/elevate-projects/02_get_table_data.sh [hunk 6]
```diff
@@ -73,4 +75,4 @@ echo ""
 sleep 2
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/02_get_table_data.sh:75]: remove one empty echo line

File: metabase/metabase-scripts/elevate-projects/03_update_json_files.sh [hunk 7]
```diff
@@ -116,4 +119,4 @@ echo ""
 sleep 2
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/03_update_json_files.sh:119]: remove one empty echo line

File: metabase/metabase-scripts/elevate-projects/04_add_question_cards.sh [hunk 8]
```diff
@@ -164,5 +167,6 @@ echo ""
 echo ""
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/04_add_question_cards.sh:167]: remove one empty echo line

File: metabase/metabase-scripts/elevate-projects/run.sh [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+echo -e "${NC}"
+# Target directory
+target_dir="."
+main_dir_path=$(pwd)
+# Loop through subdirectories
+for dir in "$target_dir"/*; do
+  # Check if it's actually a directory
+  if [[ -d "$dir" ]]; then
+    echo -e "${PINK} Started Processing Directory: $dir ${NC}"
+    # Change directory to the subdirectory
```
Comment (Vivek-M-08) [inline metabase/metabase-scripts/elevate-projects/run.sh:27]: add one empty line after Started Processing Directory:

**Review by Vivek-M-08 — CHANGES_REQUESTED** [review]

File: metabase/metabase-scripts/elevate-projects/02_get_table_data.sh (hunk 6 above)
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/02_get_table_data.sh:73]: done

File: metabase/metabase-scripts/elevate-projects/03_update_json_files.sh (hunk 7 above)
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/03_update_json_files.sh:118]: done

File: metabase/metabase-scripts/elevate-projects/run.sh [hunk 10]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# ANSI escape codes for colors
+PINK="\033[0;35m"
+GREEN="\033[0;32m"
+NC="\033[0m" # No Color (reset to default)
```
[the remark this answers is not in the document]
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/run.sh:6]: done

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh [hunk 11]
```diff
@@ -4,31 +4,31 @@
 ## retrieves relevant IDs (collection, dashboard, and database), and saves them to a file.
 ## It uses curl for API requests and jq to handle JSON responses.
 
+#External Path 
+report_path=$1
+folder_name=$(basename "$report_path")
+
+new_name=$(echo "$folder_name" | tr -s '-' ' ')
```
[the remark this answers is not in the document]
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh]: done

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh (hunk 1 above)
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh]: removed

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh (hunk 2 above)
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh]: done

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh (hunk 4 above)
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh]: removed

File: metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh (hunk 5 above)
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/01_create_dashboard.sh]: removed

File: metabase/metabase-scripts/elevate-projects/04_add_question_cards.sh (hunk 8 above)
Comment (prashanthShiksha) [reply metabase/metabase-scripts/elevate-projects/04_add_question_cards.sh]: done

**Pull request merged**